### DISCUSSION TITLE
How to restore a previously deleted organization or organization repository?

### DISCUSSION THREAD

**Discussion by yoonost** [community/community#141350] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to restore a previously deleted organization or organization repository?

**Comment by t1noo7** [comment, marked as answer]
Hi, @yoonost

As for your question, in this platform, there are 2 problems you're looking for:

1. Restoring a Deleted Organization:
Unfortunately, it is not possible to restore a deleted organization. Once an organization is deleted, all associated repositories, projects, issues, and other data are permanently removed, and GitHub does not provide a way to recover them. You would need to recreate the organization and manually recreate repositories, settings, teams, etc.

If you anticipate deleting an organization but think you might need it again in the future, archiving repositories or transferring them to a different organization or user would be a better option than deleting the organization entirely.

2. Restoring a Deleted Repository within an Organization:
If you've accidentally deleted a repository, GitHub allows you to restore deleted repositories within a limited time frame. Follow these steps:

Timeframe: You can restore a deleted repository within 90 days of deletion.
How to Restore a Deleted Repository:
Go to your GitHub organization's main page.
Click on Settings (usually at the bottom left).
Under the "Deleted Repositories" section (found on the settings page), you'll see a list of deleted repositories within the past 90 days.
Find the repository you wish to restore and click "Restore" next to its name.
Note: Restoring a repository restores the entire repository, including its history, branches, commits, issues, and pull requests. However, the repository URL will remain the same, so if it was public and has already been taken by another user, you won't be able to restore it with the same URL.

Hope this helps you better :)

**Reply by mayankrathod-mrr** [reply]
my repo is deleted from github desktop
